web/site/content/en-CA/help.md

```diff
@@ -1,33 +1,43 @@
 # Annual Report Overview
 
 ## Things to do prior to filing your Annual Report
-- Register your BC ServiceCard and install the BC ServiceCard application on your mobile phone
-- Ensure your business information is up to date. A summary will be shown in your Annual Report filing.
+- Setup your <a href="https://id.gov.bc.ca/account/" target="_blank">BC Services Card (BCSC) Account</a>. This digital ID is a secure and easy way to login to access government services. 
+- Review and confirm the registered office addresses and directors in Corporate Online.
+- If updates are needed, go to <a href="https://www.corporateonline.gov.bc.ca" target="_blank">Corporate Online</a> to update, then return to file your annual report.
+- When you file your annual report, a Basic account will be created. <a href ="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/news-updates/modernization-updates/using-the-business-registry#account"> Learn more about a Basic account </a> to ensure it is right for you. 
+- If you need a <a href ="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/news-updates/modernization-updates/using-the-business-registry#account">Premium account</a>, set it up <a href = "https://www.account.bcregistry.gov.bc.ca/choose-authentication-method">here </a> then use the link in your Annual report reminder to complete your annual report filing. 
```

Comment:
There's a mixture of opening a new tab, or going to the linked page.
It looks deliberate, as all the account setup pages are direct links